crates/ibc/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -1331,13 +1333,18 @@ mod tests {
     use test_log::test;
 
     use ibc_proto::ics23::ProofSpec as Ics23ProofSpec;
+
+    #[cfg(feature = "serde")]
```

Comment:
for this file, can we instead put all the `serde`-only items under a sub-module, and annotate just that module with `#[cfg(feature = "serde")]`?

Reply:
https://github.com/cosmos/ibc-rs/pull/331/commits/4c466468dc5b19486fd5815f4058d680deeabb80